From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Kanaka Raju Nayana <kanaka.raju.nayana@intel.com>
Date: Wed, 29 Oct 2025 06:14:41 +0530
Subject: [PATCH] platform/x86/intel: Disable unused drivers in makefile

Disable all Intel platform drivers except pmt and vsec to
focus the build scope and avoid missing source file dependencies.

This ensures the build system only attempts to compile drivers with
available source files in the backport.

Signed-off-by: Kanaka Raju Nayana <kanaka.raju.nayana@intel.com>
---
 drivers/platform/x86/intel/Makefile | 29 -------------------------
 1 file changed, 29 deletions(-)

diff --git a/drivers/platform/x86/intel/Makefile b/drivers/platform/x86/intel/Makefile
index a304202..e7a971e 100644
--- a/drivers/platform/x86/intel/Makefile
+++ b/drivers/platform/x86/intel/Makefile
@@ -15,37 +15,8 @@ obj-$(CONFIG_INTEL_PMT_CLASS)		+= pmt/
 #obj-$(CONFIG_INTEL_WMI)			+= wmi/
 #obj-$(CONFIG_INTEL_UNCORE_FREQ_CONTROL)	+= uncore-frequency/
 
-
-# Intel input drivers
-intel-target-$(CONFIG_INTEL_HID_EVENT)		+= hid.o
-intel-target-$(CONFIG_INTEL_VBTN)		+= vbtn.o
-
-# Intel miscellaneous drivers
-intel-target-$(CONFIG_INTEL_INT0002_VGPIO)	+= int0002_vgpio.o
-intel-target-$(CONFIG_INTEL_ISHTP_ECLITE)	+= ishtp_eclite.o
-intel-target-$(CONFIG_INTEL_OAKTRAIL)		+= oaktrail.o
-intel-target-$(CONFIG_INTEL_SDSI)		+= sdsi.o
 intel-target-$(CONFIG_INTEL_VSEC)		+= vsec.o
 
-# Intel PMIC / PMC / P-Unit drivers
-intel-target-$(CONFIG_INTEL_BYTCRC_PWRSRC)	+= bytcrc_pwrsrc.o
-intel-target-$(CONFIG_INTEL_BXTWC_PMIC_TMU)	+= bxtwc_tmu.o
-intel-target-$(CONFIG_INTEL_CHTDC_TI_PWRBTN)	+= chtdc_ti_pwrbtn.o
-intel-target-$(CONFIG_INTEL_CHTWC_INT33FE)	+= chtwc_int33fe.o
-intel-target-$(CONFIG_X86_ANDROID_TABLETS)	+= crystal_cove_charger.o
-intel-target-$(CONFIG_INTEL_MRFLD_PWRBTN)	+= mrfld_pwrbtn.o
-intel-target-$(CONFIG_INTEL_PUNIT_IPC)		+= punit_ipc.o
-
-# TPMI drivers
-intel-target-$(CONFIG_INTEL_PLR_TPMI)		+= plr_tpmi.o
-intel-target-$(CONFIG_INTEL_TPMI_POWER_DOMAINS)	+= tpmi_power_domains.o
-intel-target-$(CONFIG_INTEL_TPMI)		+= vsec_tpmi.o
-
-# Intel Uncore drivers
-intel-target-$(CONFIG_INTEL_RST)		+= rst.o
-intel-target-$(CONFIG_INTEL_SMARTCONNECT)	+= smartconnect.o
-intel-target-$(CONFIG_INTEL_TURBO_MAX_3)	+= turbo_max_3.o
-
 # Add 'intel' prefix to each module listed in intel-target-*
 define INTEL_OBJ_TARGET
 intel-$(1)-y := $(1).o
-- 
2.43.0

